modules/distribution/src/bin/fips.sh

```
@@ -14,373 +14,294 @@
 
-EXPECTED_BC_FIPS_CHECKSUM="c8df3d47f9854f3e9ca57e9fc862da18c9381fa9"
-EXPECTED_BCPKIX_FIPS_CHECKSUM="722eaefa83fd8c53e1fc019bde25e353258ed22b"
-EXPECTED_BCUTIL_FIPS_CHECKSUM="1d37b7a28560684f5b8e4fd65478c9130d4015d0"
+EXPECTED_BC_FIPS_CHECKSUM="061fbe8383f70489dda95a11a2a4739eb818ff2c"
+EXPECTED_BCPKIX_FIPS_CHECKSUM="41d15c70437440d63b65225d7c00873a030d25d0"
+EXPECTED_BCUTIL_FIPS_CHECKSUM="30b41ebc759a4f02e2ff7ab9acb09268923ee41f"
+EXPECTED_BCPG_FIPS_CHECKSUM="727e087a843f3a5a8143e4f3a7518c8c3517df18"
+EXPECTED_BCTLS_FIPS_CHECKSUM="d2979016bf75ef8b5e8aa17211399651a391a21f"
 
 # Get standard environment variables
 PRGDIR=`dirname "$PRG"`
```

Shellcheck (0.11.0)

[style] Use $(...) notation instead of legacy backticks `...`.

(SC2006)